### DISCUSSION TITLE
Keep blog posts from being displayed on the initial home / landing page?

### DISCUSSION THREAD

**Discussion by ZiaCreatesIdeas** [daattali/beautiful-jekyll#997] [General]
Thanks so much for the creation of Beautiful Jekyll! 
I would like the home page to not include any posts. 
I see in the FAQ page there is suggested code updating a pagination setting and copying the index.html to a new directory, however this did not work for me. Only by modifying the _layout/home.html file was I able to delete the posts, but I also removed the footer - which I enjoyed having displayed.
Any suggestions would be welcome. Thanks.

**Comment by daattali** [comment]
If I recall correctly (I haven't touched this code in years so I may be wrong), the blog posts show up on a page that has a `layout: home`. The index.html page by default has that layout. So try removing that `layout: home` line from index.html

**Reply by ZiaCreatesIdeas** [reply]
Thanks thanks thanks!! That did the trick. 'Really appreciate it. I appreciate the effort you put into creating this 'stack' and that you put all the configuration options into the yaml file. Git-pages plus Jekyll is an interesting beast. Hearing that you haven't touched this code in awhile, I'm curious what you moved on to?
Cheers!

**Reply by daattali** [reply]
I add new features and fix bugs, but I haven't touched that specific code. Regardless, Beauriful Jekyll was just a pet project from 8 years ago that wasn't meant for other people to use. I spend most of my time on my actual job and building open source R packages. Jekyll is not something I ever touch anymore except for maintenance here

**Reply by ZiaCreatesIdeas** [reply]
Very Cool! Thanks for sharing. It is an interesting pet project for sure. This was by far the best of the Jekyll themes I came across, you really put some effort and organization into it. If you're R packages are anything like Beautiful Jekyll, I'm sure your employer is very happy! I'm assuming most people have moved on from Jekyll towards CSS style templates or wix style builder sites, but Git-Pages means I don't have to sign up for a monthly fee, just to test a few ideas online.

**Reply by vonvoo** [reply]
Thank you as always, @daattali !!! Might you consider updating the faq with this info?

**Reply by daattali** [reply]
@vonvoo if you'd like to come up with the exact phrasing for a question+answer, I'd be happy to add it to the FAQ
